centrapay/centrapay-docs
# Fixed the default cover img width

The default cover image is full width responsively. 

Test plans: 
1. Go to https://docs.centrapay.com/guides/point-of-sale
2. Open developer tool
3. Toggle the device toolbar for a responsive view
4. Key in 4k width or Mobile version 
5. The default image should be full-width. 
6. Positioning below the navigation bar

For screen over 1980px:
![Screenshot 2023-02-10 at 1 41 17 PM](https://user-images.githubusercontent.com/32785419/217971608-ff1dee81-0da6-407c-bfc1-e7f082f983c0.png)


For mobile:
![Screenshot 2023-02-10 at 1 41 34 PM](https://user-images.githubusercontent.com/32785419/217971625-8cf0393c-b66c-49f9-8585-aaa7c1e6944b.png)
